app/views/ExposureHistory/ExposureCalendarView.js

```diff
@@ -61,10 +63,19 @@ export const ExposureCalendarView = ({ history, weeks }) => {
         <LegendItem riskLevel={Risk.Possible}>
           {languages.t('history.possible_exposure')}
         </LegendItem>
-        <LegendItem riskLevel={Risk.Unknown}>
-          {languages.t('label.no_data')}
-        </LegendItem>
+        {!isSpecificLang && (
```

Comment:
Adding `flex-wrap: wrap` to `<LegendRow>` should allow it to wrap if it doesnt fit in one line... Does that work?

Reply:
Good notice, thanks!